artifactory/commands/python/twine.go

```diff
@@ -194,3 +244,17 @@ func (tc *TwineCommand) isRepoConfigFlagProvided() bool {
 func (tc *TwineCommand) getRepoConfigFlagProvidedErr() string {
 	return "twine command must not be executed with the following flags: " + coreutils.ListToText(twineRepoConfigFlags)
 }
+
+func CreateAqlQueryForSearch(repo, fileInitial string) string {
+	itemsPart :=
+		`{` +
+			`"repo": "%s",` +
+			`"$or": [{` +
+			`"$and":[{` +
+			`"path": {"$match": "*"},` +
+			`"name": {"$match": "%s*"}` +
+			`}]` +
+			`}]` +
+			`}`
+	return fmt.Sprintf(itemsPart, repo, fileInitial)
+}
```

Comment:
This is not required if the same functionality can be achieved without search operation like suggested above.